articles/active-directory-b2c/active-directory-b2c-faqs.md

```diff
@@ -130,6 +130,9 @@ Not currently. This feature is on our roadmap. Verifying your domain in the **Do
 
 Follow these steps to delete your Azure AD B2C tenant:
 
+1. Create a new local user account in the B2C tenant, using the Azure Active Directory blade and assign it the Global Administrator directory profile.
+1. If the B2C tenant was linked to a subscription (which likely is associated to a non-B2C tenant), delete the Azure resource representing your B2C tenant's billing from the non-B2C tenant.
+1. All of the following steps should be completed using the newly created Global Administrator account.
```

Comment:
Maybe this only happens in a niche case which I cannot otherwise explain, but I had the Portal tell me that I have to remove all subscriptions when I was trying to delete the tenant using the initial user account which is external to the B2C directory. Using a local global admin account instead I never had that error shown.

Reply:
You are correct - that is the result of only adding to the list and not removing the original content. I believe that line should be removed completely at this point.